README.md

```diff
@@ -107,6 +107,13 @@ Appends `dpiAware` tag for custom DPI aware applications.
       on_off         (optional) true/false or 0/1. Default is 0
       per_monitor    (optional) win81 per-monitor DPI awareness. Default is 0
 
+#### GdiScaling
+
+Appends `gdiScaling` tag for enabling GDI scaling
+
+    Parameters       [on_off] [per_monitor]
```

Comment:
Is `[per_monitor]` redundant here?

Reply:
It's not, good old fashion copy and past issue.